### PR TITLE
Add create page functionality

### PR DESCRIPTION
**Pull request by ahmetrehaseker**
Notion API expects `rich_text` instead of `text` while querying the database with property filtering

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -72,7 +72,7 @@ impl NotionApi {
         Ok(Self { client })
     }
 
-    async fn make_json_request(&self, request: RequestBuilder) -> Result<Object, Error> {
+    pub async fn make_json_request(&self, request: RequestBuilder) -> Result<Object, Error> {
```
Comment (jakeswenson) [inline src/lib.rs:75]: If we make this public can we add some doc comments on it?

**Review by jakeswenson — CHANGES_REQUESTED** [review]
Thanks for the PR! I appreciate it. I think the build is failing due to a cargo fmt issue, do you know if you're seeing that on your end?

I left a comment to add a doc comment for the private method we'd be making public in this PR.

Thanks again!

File: src/lib.rs [hunk 2]
```diff
@@ -172,6 +173,21 @@ impl NotionApi {
         }
     }
 
+    pub async fn create_page<T: Into<PageCreateRequest>>(&self, page: T) -> Result<Page, Error> {
```
Comment (jakeswenson) [inline src/lib.rs:176]: Can we add a doc document to this?

```suggestion
    /// Creates a new page
    pub async fn create_page<T: Into<PageCreateRequest>>(&self, page: T) -> Result<Page, Error> {
```

**Review by jakeswenson — APPROVED** [review]
This is awesome! Thank you for all the improvements! If you don't want to add the doc comment I can before releasing? Let me know how you'd like to proceed!

**Comment by ahmetrehaseker** [comment]
Sorry, I should add WIP tag I guess, I am testing this now when I finalize the tests I will add the documentation

**Comment by ahmetrehaseker** [comment]
I am unable to remove that WIP tag, I tested the changes the PR is ready to be merged I guess.

**Comment by AusCyberman** [comment]
Can you yknow merge this?

**Review by jakeswenson — APPROVED** [review]
LGTM! Thank you for the contribution!

**Pull request merged**